src/karmen_frontend/src/components/gateways/heartbeat.js

```diff
@@ -46,24 +46,22 @@ const UpgradeModal = ({ shouldShow }) => {
     isOpen && (
       <Modal>
         <h1 className="modal-title text-center">
-          Application has been upgraded
+          Application has been upgraded, please reload the page to continue.
```

Comment:
Karmen has been updated, please reload the page to continue working.

Reply:
General question (not sure if @tibor-kulcar can answer that): what if user had some work unfinished?

Reply:
ceebcf8e67434e732085bd85c934b9d42814969f